charts/karmada/_crds/bases/autoscaling.karmada.io_cronfederatedhpalists.yaml

```diff
@@ -0,0 +1,269 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: cronfederatedhpalists.autoscaling.karmada.io
+spec:
+  group: autoscaling.karmada.io
+  names:
+    kind: CronFederatedHPAList
+    listKind: CronFederatedHPAListList
```

Comment:
This CRD should not be generated.

Reply:
I don't know why it would be generated. Do you know why?

Reply:
Just removed the file which might introduced by the typo mentioned at https://github.com/karmada-io/karmada/pull/3692#discussion_r1241070752.